homeassistant/components/garages_amsterdam/__init__.py

```diff
@@ -24,17 +23,12 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
 
     await coordinator.async_config_entry_first_refresh()
 
-    hass.data.setdefault(DOMAIN, {})
-    hass.data[DOMAIN][entry.entry_id] = coordinator
+    entry.runtime_data = coordinator
```

Comment:
Please change the type here from ConfigEntry to GaragesAmsterdamConfigEntry. Also for the unload